lab03-kaylee/fs-textData.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+const fs = require('fs');
+
+module.exports = function(callback) {
+  let fileArray = [];
+
+  fs.readFile(`${__dirname}/data/one.txt`, function(err, data) {
+    if(err) throw err;
+    let dataOne = data.toString('hex', 0, 8);
+    fileArray.push(dataOne);
+
+    fs.readFile(`${__dirname}/data/two.txt`, function(err, data) {
+      if(err) throw err;
+      let dataTwo = data.toString('hex', 0, 8);
+      fileArray.push(dataTwo);
+
+      fs.readFile(`${__dirname}/data/three.txt`, function(err, data) {
+        if(err) throw err;
+        let dataThree = data.toString('hex', 0, 8);
+        fileArray.push(dataThree);
+        callback(fileArray);
+      });
+    });
+  });
+};
+
+//Asycnhronous nested callback pattern-- a nested function won't run until the function directly above it has completely finished running.
```

Comment:
Comments are great and I definitely encourage them, but get in the habit of managing a separate branch that contains your comments and submit a clean branch to Canvas. We'll be docking off points for that next week.